### ISSUE TITLE
Links in tooltips are clickable when invisible

### ISSUE THREAD

**Issue by Sire** [chanan/BlazorStrap#29]
When tooltip disappears, the invisible link is still clickable if you hover above its location. 
Is there any better way of hiding the tooltips?

Reproduce with:

```
<Tooltip Target="div1" Placement="Placement.Bottom">
Text here. Text here. Text here. Text here.<br />
Text here. Text here. Text here. Text here.<br />
<a target="_blank" href="https://www.github.com/">Read more</a>.
</Tooltip>

```

**Comment by peterblazejewicz** [comment]
The representation of the tooltip is recreated in BS and destroyed when tooltip is no longer active.
Let me fix this via custom css rule

**Comment by peterblazejewicz** [comment]
This can be probably properly fixed that way:

```diff
--- a/src/BlazorStrap/content/blazorStrapInterop.js
+++ b/src/BlazorStrap/content/blazorStrapInterop.js
@@ -14,14 +14,24 @@
         return true;
     },
     tooltip: function (target, tooltip, arrow, placement) {
+        var instance;
         var reference = document.getElementById(target);
-        reference.addEventListener("mouseover", function () {
+        function mouseoverHandler() {
+            reference.removeEventListener("mouseover", mouseoverHandler);
+            reference.addEventListener("mouseout", mouseoutHandler);
             tooltip.className = "tooltip fade show bs-popover-" + placement;
-            showPopper(reference, tooltip, arrow, placement);
-        });
-        reference.addEventListener("mouseout", function () {
+            instance = showPopper(reference, tooltip, arrow, placement);
+        }
+        function mouseoutHandler() {
+            reference.removeEventListener("mouseout", mouseoutHandler);
+            reference.addEventListener("mouseover", mouseoverHandler);
             tooltip.className = "tooltip hide";
-        });
+            if (instance) {
+                instance.destroy && instance.destroy();
+                instance = undefined;
+            }
+        }
+        reference.addEventListener("mouseover", mouseoverHandler);
         return true;
     }
 };
@@ -48,4 +58,5 @@ function showPopper(reference, popper, arrow, placement) {
             }
         }
     );
+    return thePopper;
 }
```
Let me know if this is OK,
thanks!

**Comment by chanan** [comment]
@peterblazejewicz Thanks! Do you want to submit a PR for this, or do you want me to integrate it?

**Comment by peterblazejewicz** [comment]
sure, will close the other one, and create new one with above changes.

**Comment by chanan** [comment]
Fixed by @peterblazejewicz in 1.0.0-preview6-15
